test/k8sT/Services.go

```diff
@@ -432,6 +432,91 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	SkipContextIf(helpers.RunsWithKubeProxy, "Checks connectivity when skipping socket lb in pod ns", func() {
```

Comment:
The PR description talks about introducing this flag to use with kube-proxy free mode, but then the tests are only validating the behaviour when kube-proxy is present. Am I missing something here?

Reply:
hmm, this IS saying don't run the test if there's kube-proxy, aka run the test if there's no kube-proxy (in kube-proxy free mode), unless I'm reading `SkipContextIf` wrong?

Reply:
This workaround should not change behavior when there's no kernel support for netns_cookie, `ctx_in_hostns` would always return `true` and we won't do any skipping in that case.

Added probe when configuring node for this flag, will issue warning if kernel's lacking support.

Reply:
`SkipContextIf(helpers.RunsWithKubeProxy, ...` should be read as "skip this context if it runs with kube-proxy", so it's correct.

Reply:
OK, I was missing something, thanks! :sweat_smile: